packages/squidex/schema/schemas/research-outputs.json

```diff
@@ -213,6 +213,186 @@
           "isRequiredOnPublish": false,
           "isHalfWidth": false
         }
+      },
+      {
+        "name": "subtype",
+        "properties": {
+          "isRequired": false,
+          "isRequiredOnPublish": false,
+          "isHalfWidth": false,
+          "fieldType": "String",
+          "editor": "Dropdown",
+          "inlineEditable": false,
+          "isUnique": false,
+          "allowedValues": [
+            "ASAP annual meeting",
```

Comment:
Even though I can see where you're coming from, I don't think it is a good idea to attach our typings to this _computer-generated, ever-changing, Squidex schemas_ 😄 We've seen the schema change its structure many times.

I think it would be better if we just try to generate the typings from the GraphQL schema instead.

Additionally, I wouldn't want to do such changes as part of this PR. :)